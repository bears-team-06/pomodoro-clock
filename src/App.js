import React  from "react";
import PomodoroClock from "./PomodoroClock";
import About from "./About";
import { BrowserRouter as Router, Route, Link } from "react-router-dom";

const App = () => {
  return (
    <Router>
      <div>
        <nav>
          <ul>
            <li>
              <Link to="/">Pomodoro Clock</Link>
            </li>
            <li>
              <Link to="/about/">About</Link>
            </li>
          </ul>
        </nav>

        <Route path="/" exact component={PomodoroClock} />
        <Route path="/about/" component={About} />
      </div>
    </Router>
  );
};

export default App;
