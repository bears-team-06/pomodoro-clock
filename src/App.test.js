import { shallow } from 'enzyme'
import React from 'react'
import App from "./App";

it('matches app snapshot', () => {
    expect(shallow(<App/>)).toMatchSnapshot();
})