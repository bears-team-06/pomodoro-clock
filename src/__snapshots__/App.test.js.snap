// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches app snapshot 1`] = `
<BrowserRouter>
  <div>
    <nav>
      <ul>
        <li>
          <Link
            replace={false}
            to="/"
          >
            Pomodoro Clock
          </Link>
        </li>
        <li>
          <Link
            replace={false}
            to="/about/"
          >
            About
          </Link>
        </li>
      </ul>
    </nav>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/about/"
    />
  </div>
</BrowserRouter>
`;
